src/main.js

```diff
@@ -1,11 +1,10 @@
 import { createApp } from 'vue'
 import App from './App.vue'
 import router from "./router";
-import store from "./store";
 import './index.css';
 import InlineSvg from "vue-inline-svg";
 
-const app = createApp(App).component('inline-svg', InlineSvg).use(store).use(router);
+const app = createApp(App).component('inline-svg', InlineSvg).use(router);
```

Comment:
any reason for removing store?

Reply:
I thought we discussed on the call that we can bring back store later when we start adding dropdowns.

Reply:
Ohhh I forgot it guess
cool cool